src/components/mobile-menu-container/mobile-menu-container.module.css

```diff
@@ -16,5 +16,6 @@
     position: fixed;
     top: unset; /* make top relative to the parent */
     width: 100vw;
+    z-index: 1;
```

Comment:
Adding this back - I thought it wasn't necessary in a previous PR, but missed the search input issue.